bundles/org.openhab.voice.googletts/README.md

```diff
@@ -0,0 +1,60 @@
+# Google Cloud Text-to-Speech
+
+Google Cloud TTS Service uses the none-free Google Cloud Text-to-Speech API to convert text or Speech Synthesis Markup Language (SSML) input into audio data of natural human speech. 
+It provides 30 voices, available in multiple languages and variants and applies DeepMind’s groundbreaking research in WaveNet and Google’s powerful neural networks. 
```

Comment:
They have added many more voices. According to https://cloud.google.com/text-to-speech:

> Google Cloud Text-to-Speech offers a selection of 180+ voices across 30+ languages and variants, enabling developers to pick the voice that works best for their application.

Reply:
I used a more general wording now. Without calling on numbers.